### PR TITLE
Changes type of RTCRTPStreamStats.ssrc from string to unsigned long.

### PR DESCRIPTION
**Pull request by henbos**
Resolves issue #105. PTAL and merge @alvestrand

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -430,8 +430,8 @@
             <dl data-link-for="RTCRTPStreamStats" data-dfn-for="RTCRTPStreamStats" class=
             "dictionary-members">
               <dt>
-                <dfn><code>ssrc</code></dfn> of type <span class=
-                "idlMemberType"><a>DOMString</a></span>
+                <dfn><code>ssrc</code></dfn> of type <span class="idlMemberType"><a>unsigned
+                long</a></span>
               </dt>
               <dd></dd>
```
Comment (alvestrand) [inline webrtc-stats.html]: While you're at it, can you fill in the explanation (the part between <dd> and </dd>) with "32-bit integer value per [[RFCxxxx]]"?

File: webrtc-stats.html [hunk 2]
```diff
@@ -410,7 +410,7 @@
         </h3>
         <div>
           <pre class="idl">dictionary RTCRTPStreamStats : RTCStats {
-             DOMString     ssrc;
+             unsigned long ssrc;
```
Comment (alvestrand) [inline webrtc-stats.html]: Please align with the others, I don't think the formatting will re-align your webIDL.

Comment (henbos) [inline webrtc-stats.html]: Done

File: webrtc-stats.html (hunk 1 above)
Comment (henbos) [inline webrtc-stats.html]: Done

File: webrtc-stats.html [hunk 3]
```diff
@@ -430,10 +430,14 @@
...
               <dt>
-                <dfn><code>ssrc</code></dfn> of type <span class=
-                "idlMemberType"><a>DOMString</a></span>
+                <dfn><code>ssrc</code></dfn> of type <span class="idlMemberType"><a>unsigned long
+                </a></span>
               </dt>
-              <dd></dd>
+              <dd>
+                <p>
+                  32-bit integer value per [[RFC3550]].
```
Comment (jan-ivar) [inline webrtc-stats.html]: unsigned

Comment (henbos) [inline webrtc-stats.html]: Description updated.
(The spec says "identified by a 32-bit numeric SSRC identifier carried in the RTP header", based on it it is not clear to me whether it should be signed, unsigned or just a series of bits, but its appendix has sample code that use uint32_t :) )

**Pull request merged**